internal/job/send-emails-job.go

```diff
@@ -2,18 +2,16 @@ package job
 
 import (
 	"log"
-	"time"
 
-	"genesis-currency-api/internal/service"
+	"genesis-currency-api/pkg/interface/service"
+
 	"github.com/robfig/cron/v3"
 )
 
 // SendEmailsJob is a cron function to send emails to subscribed users.
 // It is executed every day at 9:00
-func SendEmailsJob(emailService *service.EmailService) {
-	scheduler := cron.New(cron.WithLocation(time.UTC))
-
-	_, err := scheduler.AddFunc("0 9 * * *", func() {
+func SendEmailsJob(cron *cron.Cron, emailService service.EmailService) {
```

Comment:
Here the same **`Go interfaces generally belong in the package that uses values of the interface type, not the package that implements those values`**

https://go.dev/wiki/CodeReviewComments#interfaces

```go
type EmailSender interface {
	Send() error
}
```